modules/ipi-install-firmware-requirements-for-installing-with-virtual-media.adoc

```diff
@@ -27,6 +27,7 @@ Red Hat does not test every combination of firmware, hardware, or other third-pa
 |====
 | Model | Management | Firmware versions
 
+| 15th Generation | iDRAC 9 | v6.10.30.00
 | 15th Generation | iDRAC 9 | v5.10.00.00 - v5.10.50.00 only
```

Comment:
we should probably remove this, the 5.x firmware versions has not received an update in over 6 months and the series 6.xx is now the recommended one by default for most recent generations

Reply:
Done, does this work?

Reply:
lgtm thanks!